sleep_stage/cluster_assignment/sleep_stage_cluster_classifier.py

```diff
@@ -0,0 +1,18 @@
+from sklearn.model_selection import train_test_split
+
+import shhs.polysomnography.polysomnography_reader as ps
+from sleep_stage.features.feature_classifier import load_feature_classifier
+
+
+def main():
+    edf_path = "/Users/blakemacnair/dev/data/shhs/polysomnography/edfs/shhs1"
```

Comment:
I think in general we need a unified way to handle machine paths so graders may easily plug and play. Or we use relative paths when possible if dealing with out own data files we upload.

Reply:
true true. I think I could try defining some global variables in a module that all of our other modules can import, that way we can just define all of our paths in a singular file. Also, once we get everything to a suitable stage, we could make them executables where we pass in paths as the parameter.